.semaphore/semaphore.yml

```diff
@@ -15,11 +15,12 @@ blocks:
         commands:
           - checkout
           - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+          - nvm use 10.15.1
```

Comment:
```suggestion
          - nvm use
```
this will just refer to the existing `.nvmrc`

Reply:
Nice, thanks!